chromadb/api/async_fastapi.py

```diff
@@ -449,7 +449,7 @@ async def _add(
         documents: Optional[Documents] = None,
         uris: Optional[URIs] = None,
     ) -> bool:
-        batch = (ids, embeddings, metadatas, documents, uris)
+        batch = (ids, [embedding.tolist() for embedding in embeddings] if embeddings is not None else None, metadatas, documents, uris)
```

Comment:
When is embeddings ever None here? I don't understand the defensive check because the input type is not None?